### PR TITLE
React - Animations 

### PR DESCRIPTION
**Pull request by gbasile**
Created a debug screen with some sample animations and animated the logo into the splash screen. 

| iOS  | Android |
| --- | --- |
|animations-ios| animations-android|

### CODE REVIEW

**Comment by frapontillo** [comment]
Anna Tatangelo WTF?



(LGTM)

**Comment by eduardb** [comment]
My only comment is that maybe all the values used for the animations should be extracted into constants, so it's easy to adjust them.

**Comment by gbasile** [comment]
@eduardb addressed

**Comment by takecare** [comment]
👍  neat

File: ReactNative/ReactTwitter/core/views/debug/debug-animation-scene.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ import {
 } from 'react-native'
 
 var Button = require('react-native-button')
+const IMAGE_SIZE_NORMAL = 100
+const IMAGE_SIZE_BIG = 200
+const OPACITY_NORMAL = 1
+const OPACITY_TRANSPARENT = 0
+const DELAY = 50
```
Comment (takecare) [inline ReactNative/ReactTwitter/core/views/debug/debug-animation-scene.js]: do these have units?

**Comment by danybony** [comment]
Looking good! 👍

**Pull request merged**